websites/S/Snipes/dist/metadata.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
```

Comment:
Use latest schema

Reply:
It doesn't matter bruh. $schema is only for client-side validation for example if they have plugin in vscode to validate schema.

Reply:
Yes but if they're going to use schema they might as well use the latest one or not use it at all, simple.

Reply:
🤦